cli/src/api/mod.rs

```diff
@@ -566,9 +583,12 @@ mod tests {
                     "num_dependencies": 14,
                     "packages": [
                         {
-                            "name": "ansi-red",
-                            "type": "npm",
-                            "version": "0.1.1"
+                            "package_descriptor": {
+                                "name": "ansi-red",
+                                "version": "0.1.1",
+                                "type": "npm"
+                            },
+                            "lockfile_path": null
```

Comment:
This has been [flattened](https://github.com/phylum-dev/phylum-types/pull/78/commits/7c50567398d9303e700fcef734d8b6b250daf501) now, right?

If we also use `#[skip_serializing_if = "Option::is_none"]`, we can avoid any changes to the format when no `lockfile_path` is used.

Reply:
It has been. Thanks, I put up a [PR](https://github.com/phylum-dev/phylum-types/pull/79) to add that attribute